Character.cpp

```diff
@@ -27,7 +27,9 @@ void Character::attack( Character& other )
     isDefending = false;
     std::cout << getName() << " has attacked " << other.getName() << std::endl;
     //subtract attackDamage from other->hitPoints
-    if( other.takeDamage(attackDamage) <= 0 )
+    other.hitPoints -= this->attackDamage;
+
+    if(other.takeDamage(attackDamage) <= 0 )
```

Comment:
`takeDamage()` performs the subtraction.  Look at its implementation.   
- [ ] remove line 30